GalaxyZooTouchTable/GalaxyZooTouchTable/Models/TableUserFactory.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Windows.Media.Imaging;
+
+namespace GalaxyZooTouchTable.Models
+{
+    enum UserType
+    {
+        Person,
+        Star,
+        Earth,
+        Light,
+        Face,
+        Heart
+    }
+
+    abstract class TableUser
+    {
+        public abstract string ThemeColor { get; }
+        public abstract BitmapImage StartButton { get; }
+    }
+
+    class PersonUser : TableUser
```

Comment:
As the code is right now, someone consuming this could instantiate their own `PersonUser` or `StarUser` etc without using the factory method. I think for all these subclasses you may want to add a private constructor like `private PersonUser() {}`. That way consumers of these models have to use the factory method.

Reply:
Yes. This makes total sense. Good eye.